proc/threads.c

```diff
@@ -454,7 +453,8 @@ int threads_timeintr(unsigned int n, cpu_context_t *context, void *arg)
 
 	hal_spinlockClear(&threads_common.spinlock, &sc);
 
-	return EOK;
+	/* Invoke scheduler */
+	return 1;
```

Comment:
This invokes sheduler only on cpu 0 due to early return earlier.

Reply:
Missed that, fixed, thanks!